Bus Scheduler/src/components/atoms/button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import '@/components/atoms/button/style.scss';
+
+type ButtonType = {
```

Comment:
I think would be good to rename this, becuase the type holds other properties than type